sda-auth/keys/c4gh_key.sec.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN CRYPT4GH ENCRYPTED PRIVATE KEY-----
+YzRnaC12MQAGc2NyeXB0ABQAAAAAq5Zoea6wEWm2lT5gad3aZQARY2hhY2hhMjBf
+cG9seTEzMDUAPBfY7N9POe/m4L+x1Z1QYEhv7x3HOU47G+wqRFVXq2WBHfrrQNHp
+1YIpd3rGNnXI8O5WoB+qZ8ynAIn2tA==
+-----END CRYPT4GH ENCRYPTED PRIVATE KEY-----
```

Comment:
```suggestion
```
Don't commit public/private keys to the repo.
If they are needed for testing have a script or container generate them dynamically.